cmd/server-startup-msg.go

```diff
@@ -158,7 +158,7 @@ func printEventNotifiers() {
 		return
 	}
 
-	arns := globalEventNotifier.GetARNList(true)
+	arns := globalEventNotifier.GetARNList()
```

Comment:
so, from now on, we will print all the targets irrespective of if it is active or not? @vadmeste

Reply:
I think it should be okay, this is just banner text